doc/source/whatsnew/v1.3.0.rst

```diff
@@ -244,6 +244,8 @@ I/O
 - Allow custom error values for parse_dates argument of :func:`read_sql`, :func:`read_sql_query` and :func:`read_sql_table` (:issue:`35185`)
 - Bug in :func:`to_hdf` raising ``KeyError`` when trying to apply
   for subclasses of ``DataFrame`` or ``Series`` (:issue:`33748`).
+- Bug in :func:`_sqlalchemy_type` that mapped pandas integer types to non-optimal SQLAlchemy integer types (:issue:`35076`)
```

Comment:
This should be an enhancement and should just mention that impacts `to_sql`

Reply:
Thanks, updated it